Update routing instructions for Oracle AI Database access (#128631)
Azure/azure-content: articles/oracle/oracle-db/oracle-database-network-plan.md

>These UDRs should define the specific destination IP prefixes and set the next hop to the appropriate NVA/firewall in the hub.  
> Without these routes, outbound traffic may bypass required inspection paths or fail to reach the intended destination.

> [!Note]

> > If you want to configure a route table (UDR route) to control the routing of packets through a network virtual appliance or firewall destined to an Oracle AI Database@Azure instance from a source in the same virtual network or a peered virtual network, the UDR prefix must be more specific or equal to the delegated subnet size of the Oracle AI Database@Azure. If the UDR prefix is less specific than the delegated subnet size, it isn't effective.

Reviewer: When routing traffic through a network virtual appliance or firewall from sources in the same virtual network or a peered virtual network, configure the user-defined route (UDR) with a prefix that is equal to or more specific than the delegated subnet size of the Oracle AI Database@Azure instance. Do not use a UDR prefix that is less specific than the delegated subnet size, because such routes won't take effect.

Reviewer 2: @taylorrees1 Can you respond to the requested changes or let us know if you would like us to commit them?

@jjaygbay1 - When the changes are ready for publication, adding a `#sign-off` comment is the best way to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

> Without these routes, outbound traffic may bypass required inspection paths or fail to reach the intended destination.

> [!Note]

> > When routing traffic through a network virtual appliance or firewall from sources in the same virtual network or a peered virtual network, configure the user-defined route (UDR) with a prefix that is equal to or more specific than the delegated subnet size of the Oracle AI Database@Azure instance. Do not use a UDR prefix that is less specific than the delegated subnet size, because such routes won't take effect.

Reviewer 2: ```suggestion
> When routing traffic through a network virtual appliance or firewall from sources in the same virtual network or a peered virtual network, configure the user-defined route (UDR) with a prefix that is equal to or more specific than the delegated subnet size of the Oracle AI Database@Azure instance. Don't use a UDR prefix that is less specific than the delegated subnet size, because such routes won't take effect.
```
fixing style
